File: Easy-Array/Sortedornot.java
import java.util.Scanner;

class Sortedornot
{
    public static boolean isSorted(int[] arr)
    {

        // Traverse through all array elements
        for (int i = 0; i < arr.length - 1; i++)
        {
            // Check if there is any element smaller than its next element
            if(arr[i]>arr[i+1])
            {
                return false;
            }
            // If any element found smaller than its next element, then array is not sorted
            else
            {
               break;
            }
        }
        return true;
    }
    public static void main(String[] args)
    {
       Scanner sc=new Scanner(System.in);
       System.out.println("Size of Array");
       int n=sc.nextInt();
       int arr[]=new int[n];
       System.out.println("Enter the elements of array");
       for(int i=0;i<n;i++)
       {
        arr[i]=sc.nextInt();
       }
       if(isSorted(arr))
       {
         System.out.println("Array is sorted");
       }
       else
       {
         System.out.println("Array is not sorted");
       }
    }
}

File: Easy-Array/Sortedornot.py
# Function to check if an array is sorted

def is_sorted(arr):
    for i in range(len(arr) - 1):
        if arr[i] > arr[i + 1]:
            return False
    return True

number_of_elements = int(input("Enter the number of elements in the array: "))
array = []
print("Enter the elements of the array:")
for _ in range(number_of_elements):
    element = int(input())
    array.append(element)
if is_sorted(array):
    print("The array is sorted.")
else:
    print("The array is not sorted.")

